content/posts/android-tv-14.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Android TV version 0.14
+subtitle: Quick, connect!
+author: Niels van Velzen
+githubusername: nielsvanvelzen
+date: 2022-08-02
+justify: center
+---
+
+<!-- markdownlint-disable MD033 MD036 -->
+
+Introducing Quick Connect, external trailers and improved crash reporting. We're back with another update.
+
+<!--more-->
+
+I'm proud to announce the new version of Jellyfin for Android TV. In this development cycle a lot of changes made it into the app, including some new features. We've saw new [contributors](#contributors) helping out and fixed a bunch of issues reported by our awesome community. In this post I'll talk about our most significant changes. A complete list of all changes is available in the [changelog](#changelog).
+
+Enjoy this update and I'll see you in the next one!
+
+\- Niels
+
+## Minimum server version changed to 10.8.1
+
+This version of the app only supports version **10.8.1** (not 10.8.0!) or newer. By dropping support for the 10.7 range of releases we were able to add new features mentioned in this post like QuickConnect and the new crash reporting. As always we advice to use the latest version of Jellyfin to get the latest bugfixes, features and security updates.
+
+Go to the [download](https://jellyfin.org/downloads/) page or read the [installing](https://jellyfin.org/docs/general/administration/installing.html) documentation to start updating.
+
+## Improved login
+
+It's now a lot easier to get started with Jellyfin on your television. Using Quick Connect you no longer need to enter a password. Instead, you'll be prompted to open Jellyfin on another device like your phone or computer to enter a 6 digit code. This way of signing in is an alternative for passwords while keeping your server secure.
+
+Another big change in the login part of the app are the redesigned screens when adding a server or user. The screens are now easier to read and use. By using bigger text and a better alignment for the on-screen keyboard. Google TV users will now see the keyboard on the right side of the television, making it possible to see connections errors without closing the keyboard.
+
+![Screenshot of the new login design](/images/posts/android-tv-14/login-1.png)
```

Comment:
Would be cool if the screenshot showed Quick Connect and a splash screen but our demo server doesn't have those. I don't have the capacity to setup a local server for this purpose either.

Reply:
I can make that happen.

Reply:
That would be awesome, I can retake the screenshot then